node/src/Pos/Wallet/Web/State/State.hs

```diff
@@ -127,6 +128,10 @@ askWalletDB = view (lensOf @WalletDB)
 -- | The 'WalletDbWriter' constraint encapsulate the fact we can write an updated
 -- copy of the 'WalletStorage', but that we cannot read it back, if not as part
 -- of an atomic update API.
+--
+-- Note that this is not a counterpart to 'WalletDbReader'. 'WalletDbReader'
```

Comment:
@nc6 Do you think this is confusing enough for us to want `WalletDbReader` to be renamed into something else? 😉

Reply:
Possibly, but I couldn't think of an immediately better name. It will be fairly easy to change if desired in the future; it's only referenced in four files or so.